serde-support/src/main/java/io/micronaut/serde/support/deserializers/ErrorCatchingDeserializer.java

```diff
@@ -55,6 +66,9 @@ public T deserialize(Decoder decoder, DecoderContext context, Argument<? super T
         } catch (IOException e) {
             throw e;
         } catch (Exception e) {
+            if (JACKSON_EXCEPTION_CLASS != null && JACKSON_EXCEPTION_CLASS.isInstance(e)) {
```

Comment:
Can we have this configurable?

Reply:
I think we should find another solution anyway, I made an issue for it https://github.com/micronaut-projects/micronaut-serialization/issues/1212